src/main/java/se/diabol/jenkins/pipeline/domain/Pipeline.java

```diff
@@ -114,7 +119,61 @@ public void setChanges(List<Change> changes) {
         return changes;
     }
 
+    @Exported
+    public long getTotalBuildTime() {
+        return totalBuildTime;
+    }
 
+    public void setTotalBuildTime() {
```

Comment:
Please rename this since it is not a setter method. Suggestion is calculateTotalBuildTime()